docs/gcp-projects.md

```diff
@@ -0,0 +1,57 @@
+# Restore snapshots from GCP across projects
+
+This guide addresses confusion regarding [github issue](https://github.com/vmware-tanzu/velero/issues/1268).
+These steps are heavily inspired by the [gcp documentation](https://cloud.google.com/compute/docs/images/sharing-images-across-projects).
+
+Assume the following...
+
+Project A [project-a]: GCP project we want to restore TO
```

Comment:
this doesn't actually get formatted as a list/separate lines - maybe just turn it into a bulleted list (add `- ` at the beginning of the line)?